### DISCUSSION TITLE
Looking for PDF Books

### DISCUSSION THREAD

**Discussion by isadora121** [community/community#29328] [Other Feature Feedback, Questions, & Ideas]
Hello there,
I am new to this forum and want to download some of my favorite books in pdf form. Please can anybody guide me to the [best pdf website](https://pdfhour.com/) for this purpose?
Thanks in advance.

**Comment by hh24k** [comment]
[book4you](https://book4you.org/) - I wouldn't say it's the best pdf website, because it more than that ... as said, it's "The world's largest ebook library"
